plugins/modules/zos_script.py

```diff
@@ -96,6 +96,20 @@
 extends_documentation_fragment:
   - ibm.ibm_zos_core.template
 
+attributes:
+  action:
+    support: full
+    description: Indicates this has a corresponding action plugin so some parts of the options can be executed on the controller.
+  async:
+    support: full
+    description: Supports being used with the ``async`` keyword.
+  check_mode:
+    support: none
+    description: Can run in check_mode and return changed status prediction without modifying target, if not supported the action will be skipped.
+  diff_mode:
+    support: none
+    description: Will return details on what has changed (or possibly needs changing in check_mode), when in diff mode.
+
 notes:
```

Comment:
I don't want to seem stupid, but I don't5 see where the new 'attributes' are being used.  Is this part of Ansible I don't know about, or is implementation code missing?